src/lando/main/management/commands/start_landing_worker.py

```diff
@@ -46,11 +46,23 @@ def _handle(self, worker: Worker, repo_type: str):
             # Check if any associated repos are unsupported, raise exception if so.
             repo.raise_for_unsupported_repo_scm(repo_type)
             scm = repo.get_scm()
-            scm.prepare_repo(repo.pull_path)
+            repos_to_handle = []
+            try:
+                scm.prepare_repo(repo.pull_path)
+                repos_to_handle.append(repo)
```

Comment:
This line should be outside the `try` block.

Reply:
Ah, yes. We only want to execute it when the `prepare_repo` has been successful, but that can go into an `else` clause instead.